presto-main/src/main/java/com/facebook/presto/memory/QueryContext.java

```diff
@@ -413,9 +420,11 @@ private void enforceUserMemoryLimit(long allocated, long delta, long maxMemory)
     @GuardedBy("this")
     private void enforceTotalMemoryLimit(long allocated, long delta, long maxMemory)
     {
+        long newTotal = allocated + delta;
```

Comment:
nit: what about using `totalMemory`  as variable name and also use it in line 424?

Reply:
Oops, I meant to reuse it. Will rename it.